include/common/mpmc_queue.h

```diff
@@ -26,12 +26,10 @@
 
 namespace grppi{
 
-constexpr int DEFAULT_SIZE = 100;
-
 template <typename T>
 class mpmc_queue{
    private:
-      int size;
+      int size = 100;
```

Comment:
why does this have a default value of 100.

Please, revise proposed resolution of issue #85.

Also note that all constructors are setting a value for size. So this seems useless.